angular-mocks/index.d.ts

```diff
@@ -132,8 +132,9 @@ declare module 'angular' {
 
     /**
       * Verifies that all of the requests defined via the expect api were made. If any of the requests were not made, verifyNoOutstandingExpectation throws an exception.
+      * @param digest Do digest before checking expectation. Pass anything expect false to trigger digest.
```

Comment:
If this is merged in spite of my protest:
* Can we indicate here that this is a "private" feature which is not actually meant to be used in normal client code? If angular purposely chose to leave it undocumented, that is good information for users to know.
* Typo: "Pass anything ~expect~ **except** false..."